webpack.config.babel.js

```diff
@@ -126,7 +126,7 @@ export default {
       inject: false,
       minify: htmlMinifyOptions,
       title: "Lockbox Entries",
-      icon: "../icons/lock.png",
+      icon: "../icons/lb_locked.svg",
```

Comment:
@changecourse Should this be the locked or unlocked icon? You can only see the entries when you're unlocked, but the locked icon is the main logo.

Reply:
>Should this be the locked or unlocked icon? You can only see the entries when you're unlocked, but the locked icon is the main logo.

@jimporter @changecourse I went ahead and changed the favicon to the unlocked icon for consistency. It's easy enough to switch back or replace entirely if we want...